src/rp2/abstract_entry_set.py

```diff
@@ -24,6 +24,8 @@
 from rp2.out_transaction import OutTransaction
 from rp2.rp2_error import RP2TypeError, RP2ValueError
 
+T = TypeVar("T", bound="AbstractEntrySet")
```

Comment:
Can you use a descriptive name for `T` (something like `AbstractEntrySetSubclass`)?

Reply:
Updated